$NetBSD: patch-ac,v 1.3 2008/12/30 15:03:03 he Exp $

This corresponds to http://rt.cpan.org/Public/Bug/Display.html?id=41060
Can't call method "isa" without a package or object reference.

--- lib/ExtUtils/MakeMaker.pm.orig	2008-10-20 20:18:35.000000000 +0200
+++ lib/ExtUtils/MakeMaker.pm
@@ -9,6 +9,7 @@ require Exporter;
 use ExtUtils::MakeMaker::Config;
 use Carp ();
 use File::Path;
+use UNIVERSAL::isa;
 
 our $Verbose = 0;       # exported
 our @Parent;            # needs to be localized
@@ -122,7 +123,7 @@ sub _verify_att {
 
         my @sigs   = ref $sig ? @$sig : $sig;
         my $given  = ref $val;
-        unless( grep { $given eq $_ || ($_ && eval{$val->isa($_)}) } @sigs ) {
+        unless( grep { $given eq $_ || ($_ && eval{UNIVERSAL::isa($val,$_)}) } @sigs ) {
             my $takes = join " or ", map { _format_att($_) } @sigs;
 
             my $has = _format_att($given);
@@ -1708,7 +1709,7 @@ If set to 'none', no man pages will be i
 =item INSTALLVENDORSCRIPT
 
 Used by 'make install' which copies files from INST_SCRIPT to this
-directory if INSTALLDIRS is set to is set to vendor.
+directory if INSTALLDIRS is set to vendor.
 
 =item INST_ARCHLIB
 
